arviz/plots/posteriorplot.py

```diff
@@ -117,8 +117,10 @@ def plot_posterior(
     backend: str, optional
         Select plotting backend {"matplotlib","bokeh"}. Default "matplotlib".
     backend_kwargs: bool, optional
-        These are kwargs specific to the backend being used. For additional documentation
-        check the plotting method of the backend.
+        These are kwargs specific to the backend being used, passed to
+
+        :func:`matplotlib.pyplot.subplots` or
+        :func:`bokeh.plotting.figure`
     show: bool, optional
         Call backend show function.
     **kwargs
```

Comment:
this argument should also have clickable references when indicating where the kwargs are passed to

Reply:
um which argument?

the part below **kwargs you mean?

Reply:
I made some changes, I think it's what you were talking about, if not please let me know and sorry for the trouble!
Hope it helps!